thoth/metrics_exporter/jobs/db.py

```diff
@@ -123,3 +125,14 @@ def check_is_schema_up2date_for_components(cls) -> None:
             else:
                 # component is using same revision head as in the database
                 metrics.graph_db_component_revision_check.labels(component_name, Configuration.DEPLOYMENT_NAME).set(0)
+
+    @classmethod
+    @register_metric_job
+    def set_last_solver_datetime(
+        cls, os_name: Optional[str] = None, os_version: Optional[str] = None, python_version: Optional[str] = None
+    ) -> None:
+        """Get datetime of the last solver synced in the database."""
+        last_solver_datetime = cls.graph().get_last_solver_datetime(
+            os_name=os_name, os_version=os_version, python_version=python_version
+        )
+        metrics.graphdb_last_solver_datetime.labels(format_datetime(last_solver_datetime)).inc()
```

Comment:
let's explicitly set to 1: `metrics.graphdb_last_solver_datetime.labels(format_datetime(last_solver_datetime)).set(1)` to keep it fixed.

Reply:
Thanks, added in https://github.com/thoth-station/metrics-exporter/pull/830/commits/31683d1e8d0876730d55d2a849dcc83154228b4b and also in #836